package.json

```diff
@@ -82,11 +82,13 @@
   "devDependencies": {
     "@commitlint/cli": "8.x",
     "@commitlint/config-conventional": "7.x",
+    "@jest/transform": "25.x",
+    "@jest/types": "25.x",
     "@types/babel__core": "7.x",
     "@types/buffer-from": "latest",
     "@types/cross-spawn": "latest",
     "@types/fs-extra": "latest",
-    "@types/jest": "23.x",
+    "@types/jest": "25.x",
```

Comment:
Is this still required now that the definitions come along with jest itself?

Reply:
Yes I believe it only moved internals into jest itself. So public API stuff still requires it. Removing it throws the following:
```
src/util/testing.ts:7:60 - error TS2503: Cannot find namespace 'jest'.

7 interface MockWithArgs<T extends MockableFunction> extends jest.MockInstance<ReturnType<T>, ArgumentsOf<T>> {
                                                             ~~~~

src/util/testing.ts:13:5 - error TS2503: Cannot find namespace 'jest'.

13   ? jest.MockInstance<R, ConstructorArgumentsOf<T>>
       ~~~~
```